### PR TITLE
Bring back HBL-USM

### PR DESCRIPTION
**Pull request by lifehackerhansol**
**Description**

We're all about reusing ancient parts of the guide XD

PicHaxx root layout image was removed. This needs to be recreated with the new steps. Old steps were slightly rewritten to accommodate the modern instructions of USM.

A warning was added to the USM page in the DSiWare section as that is where this guide will be most relevant. (This could be worded better, please leave a reply if you know what to do here.)

Page is untested because Ruby for whatever reason isn't cooperating with me on my PC.

### CODE REVIEW

**Comment by lifehackerhansol** [comment]
The instructions are fine, I'm more worried about the warnings we should add to relevant sections where users should follow this particular guide. i.e. where to put it, how to word it

**Comment by lifehackerhansol** [comment]
Some rewrites to the troubleshooting page to align with new pichaxx page. Linked troubleshooting step for USM directly in the USM page.

Made it the main guide path (this is still in debate, but it's there for now. please don't merge yet)

**Comment by lifehackerhansol** [comment]
No longer main guide path.

PR _should_ be ready? I'm not certain. Please double check.

**Comment by lifehackerhansol** [comment]
Hardware button warnings were moved to right under the SAFE_MODE section of relevant page as discussed.

File: _pages/en_US/homebrew-launcher-(pichaxx).txt [hunk 1]
```diff
@@ -20,10 +17,8 @@ This process will overwrite your Pokémon Picross save file, if you have one. If
   + You can scan [this QR code](http://api.qrserver.com/v1/create-qr-code/?color=000000&bgcolor=FFFFFF&data=ESHOP://50010000037815&margin=0&qzone=1&size=400x400&ecc=L) using the Nintendo 3DS Camera for a direct link to the eShop app
   + Your SD card must be inserted in your device to install Pokémon Picross
 * Your `movable.sed` file from completing [Seedminer](seedminer)
-* The latest release of [SafeB9SInstaller](https://github.com/d0k3/SafeB9SInstaller/releases/download/v0.0.7/SafeB9SInstaller-20170605-122940.zip) (direct download)
-* The latest release of [boot9strap](https://github.com/SciresM/boot9strap/releases/download/1.4/boot9strap-1.4.zip) (direct download)
 * The latest release of [Luma3DS](https://github.com/LumaTeam/Luma3DS/releases/latest) (the Luma3DS `.zip` file)
-* The latest release of [universal-otherapp](https://github.com/TuxSH/universal-otherapp/releases/latest) (`otherapp.bin`)
+* The latest release of [unSAFE_MODE](https://github.com/zoogie/unSAFE_MODE/releases/latest)
```
Comment (lilyuwuu) [inline _pages/en_US/homebrew-launcher-(pichaxx).txt:21]: add "(the RELEASE `.zip` file)"

File: _pages/en_US/homebrew-launcher-(pichaxx).txt [hunk 2]
```diff
@@ -67,26 +57,10 @@ This process will overwrite your Pokémon Picross save file, if you have one. If
 1. Reinsert your SD card into your device
 1. Power on your device
 1. Launch "Pokémon Picross"
-1. If the exploit was successful, your device will have booted into SafeB9SInstaller
+1. If the exploit was successful, your device will have booted into the Homebrew Launcher
   + If you get an error message, [follow this troubleshooting guide](troubleshooting#installing-boot9strap-pichaxx)
```
Comment (lilyuwuu) [inline _pages/en_US/homebrew-launcher-(pichaxx).txt:61]: Troubleshooting link needs to be updated

File: _pages/en_US/troubleshooting.txt [hunk 3]
```diff
@@ -146,7 +128,11 @@ If this is unsuccessful, try using another SD card.
 
 ### DSiWare Management menu crashes without showing USM menu
 
-Ensure that `F00D43D5.bin` is the only file in `Nintendo 3DS` -> `<ID0>` -> `<ID1>` -> `Nintendo DSiWare`. If it is, then re-create it with the [unSAFE_MODE Exploit Injector](https://3ds.nhnarwhal.com/3dstools/unsafemode.php). If this fails, then custom firmware may have been uninstalled on this device in a way that makes this method impossible to perform. If this is the case, join [Nintendo Homebrew on Discord](https://discord.gg/MWxPgEp) for assistance.
+Ensure that `F00D43D5.bin` is the only file in `Nintendo 3DS` -> `<ID0>` -> `<ID1>` -> `Nintendo DSiWare`. If it is, then re-create it with the [unSAFE_MODE Exploit Injector](https://3ds.nhnarwhal.com/3dstools/unsafemode.php). 
+
+Also, ensure that `usm.bin` exists on the root of the SD card. If it does not exist, then download the latest release of [unSAFE_MODE](https://github.com/zoogie/unSAFE_MODE/releases/latest) (the RELEASE `.zip` file), and copy the `usm.bin` inside to the root of your SD card.
```
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:133]: Change to

"Also, ensure that `usm.bin` is on the root of the SD card. If it is missing, then download the latest release of [unSAFE_MODE](https://github.com/zoogie/unSAFE_MODE/releases/latest) (the RELEASE `.zip` file) and copy `usm.bin` to the root of your SD card."

We could also just directly link to the usm.bin on the repo

**Review by lilyuwuu — COMMENTED** [review]
Also, Homebrew Launcher (PicHaxx) and Installing boot9strap (HBL-USM) need to be added to `_data/navigation/en_US.yml` and `/assets/js/_main.js` or else the table of contents will probably break

File: _pages/en_US/homebrew-launcher-(pichaxx).txt [hunk 4]
```diff
@@ -20,10 +17,8 @@ This process will overwrite your Pokémon Picross save file, if you have one. If
   + You can scan [this QR code](http://api.qrserver.com/v1/create-qr-code/?color=000000&bgcolor=FFFFFF&data=ESHOP://50010000037815&margin=0&qzone=1&size=400x400&ecc=L) using the Nintendo 3DS Camera for a direct link to the eShop app
   + Your SD card must be inserted in your device to install Pokémon Picross
 * Your `movable.sed` file from completing [Seedminer](seedminer)
-* The latest release of [SafeB9SInstaller](https://github.com/d0k3/SafeB9SInstaller/releases/download/v0.0.7/SafeB9SInstaller-20170605-122940.zip) (direct download)
-* The latest release of [boot9strap](https://github.com/SciresM/boot9strap/releases/download/1.4/boot9strap-1.4.zip) (direct download)
 * The latest release of [Luma3DS](https://github.com/LumaTeam/Luma3DS/releases/latest) (the Luma3DS `.zip` file)
-* The latest release of [universal-otherapp](https://github.com/TuxSH/universal-otherapp/releases/latest) (`otherapp.bin`)
+* The latest release of [unSAFE_MODE](https://github.com/zoogie/unSAFE_MODE/releases/v1.3)
```
[the remark this answers is not in the document]
Comment (lifehackerhansol) [reply _pages/en_US/homebrew-launcher-(pichaxx).txt]: Already resolved.

File: _pages/en_US/troubleshooting.txt (hunk 3 above)
Comment (lifehackerhansol) [reply _pages/en_US/troubleshooting.txt:133]: Went with your first suggestion.

**Review by lilyuwuu — APPROVED** [review]
lgtm

**Pull request merged**